File: code/richediting/src/main/kotlin/org/fbme/ide/richediting/CustomFacet.java
package org.fbme.ide.richediting;

import jetbrains.mps.extapi.module.ModuleFacetBase;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.mps.openapi.module.SModule;

public class CustomFacet extends ModuleFacetBase {

    private static final String FACET_TYPE = "library";

    protected CustomFacet(@NotNull SModule module) {
        super(FACET_TYPE, module);
    }
}

File: code/platform/src/main/kotlin/org/fbme/ide/platform/CustomFacet.kt
package org.fbme.ide.platform

import jetbrains.mps.extapi.module.ModuleFacetBase
import org.jetbrains.mps.openapi.module.SModule

class CustomFacet(module: SModule) : ModuleFacetBase(FACET_TYPE, module) {
    companion object {
        private const val FACET_TYPE = "library"
    }


}
